<launch>

    <arg name="yaw" />
    <arg name="size_x" />
    <arg name="size_y" />
    <arg name="place_x" />
    <arg name="place_y" />

    <arg name="obs_path_box_seg" />

    <arg name="obstacle_file_path" />

    <!-- <arg name="yaw" default="0.0" />
    <arg name="size_x" default="10.0" />
    <arg name="size_y" default="5.0" />
    <arg name="place_x" default="12.0" />
    <arg name="place_y" default="0.0" /> -->

    <!-- <arg name="obstacle_file_path" value="$(find hollow_create)/obstacles.json" /> -->

    <node pkg="hollow_create" name="obstacles_create" type="obstacles_obtain.py" >
        <param name="obstacle_file_path" type="string" value="$(arg obstacle_file_path)" />
    </node>

    <node pkg="hollow_create" name="obstacles_display" type="hollow_create" >
        <param name="yaw" type="double" value="$(arg yaw)" />
        <param name="size_x" type="double" value="$(arg size_x)" />
        <param name="size_y" type="double" value="$(arg size_y)" />
        <param name="place_x" type="double" value="$(arg place_x)" />
        <param name="place_y" type="double" value="$(arg place_y)" />

        <param name="obs_path_box_seg" type="int" value="$(arg obs_path_box_seg)" />
    </node>


</launch>
